packages/core/content-releases/admin/src/services/release.ts

```diff
@@ -273,7 +275,25 @@ const releaseApi = createApi({
             method: 'DELETE',
           };
         },
-        invalidatesTags: () => [{ type: 'Release', id: 'LIST' }],
+        invalidatesTags: () => [
+          { type: 'Release', id: 'LIST' },
+          { type: 'EntriesInRelease', id: 'LIST' },
+        ],
+      }),
+      getMappedEntriesInReleases: build.query<
+        MapEntriesToReleases.Response,
+        MapEntriesToReleases.Request['query']
+      >({
+        query(params) {
+          return {
+            url: '/content-releases/mapEntriesToReleases',
+            method: 'GET',
+            config: {
+              params,
+            },
+          };
+        },
+        providesTags: [{ type: 'EntriesInRelease', id: 'LIST' }],
```

Comment:
If you're not ever getting an individual one, you don't need to supply an `id` you can just supply the string as the tag 👍🏼